src/test/java/org/cryptomator/cryptofs/fh/OpenCryptoFileTest.java

```diff
@@ -93,19 +93,20 @@ public void testCloseImmediatelyIfOpeningFirstChannelFails() {
 	}
 
 	@Test
-	@DisplayName("Opening a file channel with TRUNCATE_EXISTING sets the file size to 0")
+	@DisplayName("Opening a file channel with TRUNCATE_EXISTING calls truncate(0) on the cleartextChannel")
 	public void testFileSizeZerodOnTruncateExisting() throws IOException {
```

Comment:
Method name misleading, since you removed the file size verification.

I assume file size is implicitly zeroed in the truncate call?

Reply:
> I assume file size is implicitly zeroed in the truncate call?

Yes, but this is not the concern of this unit test any more. I renamed the method in https://github.com/cryptomator/cryptofs/pull/270/commits/38135c935e5da7c5bd55dfd6ea3ef70425257865